File: Codes/Dalily_Flash/Week_3/28Jan/Java/Program2.java
import java.util.*;

class InputFactorial {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);
		System.out.print("Enter First = ");
		int first = sc.nextInt();
		System.out.print("Enter Last = ");
		int last = sc.nextInt();
		for(int i=first;i<=last;i++) {  
			int fact = 1;
			for(int no=1;no<=i;no++) {

				fact = fact*no;
			}
			System.out.println("Factorial of "+i+" = "+fact);
		}
	}
}

File: Codes/Dalily_Flash/Week_3/28Jan/Python/Program2.py
first = int(input("Enter Start = "))
last = int(input("Enter End = "))
for no in range(first,last+1):
    fact = 1
    for no1 in range(1,no+1):
        fact = fact*no1
    print("Factorial of ",no,"=",fact)

